Handle removing user

danil-topc opened the pull request:
#34 
- Remove user from all ldap groups
- Delete user's chat account

matejak commented:
it doesn't work for me for some reason in connection with the `entint` setup - I get error boxes, and the browser informed me that  the URL of `api/ldap/users/` doesn't support the `DELETE` method. 
Could you please check it in a container?

danil-topc commented:
@matejak re-checked it on fresh container setup. My fault, issue was that I checked deletion only on users list and it worked, forgot that we have deletion on user detail page. Fixed it.
I had to remove "delete empty groups?" checkbox and feature, because it was outdated, after we switched to users/groups management from nextcloud api to ldap. Should I add it in new PR ?

matejak commented:
No, that's fine for the time being. Thank you for the PR!
